### ISSUE TITLE
Feature: Timing ranges/arrows on top of the diagram

### ISSUE THREAD

**Issue by darkstar** [drom/wavedrom#205]
It would be neat to have a feature that allowed diagrams similar to that one:
![image](https://user-images.githubusercontent.com/43795/45591177-45d3e300-b94b-11e8-98ad-46d672bb5785.png)
Note the "T_s", "T_c", "T_{s0}", etc. labels and arrows at the top

**Comment by clobybob28** [comment]
Any chance of this being implemented? I also see #51 which sounds similar, but I can't see the full conversation. Looks like it's an older issue from Google Code.

**Comment by crdavis12** [comment]
Any chance of this being implemented? I also see #51 which sounds similar, but I can't see the full conversation. Looks like it's an older issue from Google Code.

**Comment by drom** [comment]
Are you looking for Arrows?
https://wavedrom.com/tutorial.html#arrows

**Comment by crdavis12** [comment]
No, I'm thinking more like dimensions on a drawing. The T_s and other labels at the top of @darkstar's diagram, that show time (or some other unit) between two events in the diagram.

**Comment by jason-s** [comment]
You can do that now, just put a blank row:

```
{signal:  
   [
     {name: 'timing', node:'..CD....E.'},
   {name:'S1',wave:'1..0.......',node:'...B..'},
    {name:'S2',wave:'1.0.....1..',node:'..A.....F.'},
  ],
 config: {hscale:1},
 head:{
   tick:null,
 },
 edge:['A-C','B-D','C<->D T1',
       'E-F', 'D<->E T2'
 ],
}
```

![image](https://user-images.githubusercontent.com/738893/92968165-7615c580-f42f-11ea-9a54-e310d929c9c8.png)

It does require using up extra timing nodes though.

Maybe it would help to be able to specify the intersection of a node and a signal, for example `A-timing@A` rather than `A-C` and having to create an artificial node C that is in the same horizontal position as A but in another waveform's row.

**Comment by adrienkohlbecker** [comment]
This MR might be helpful https://github.com/wavedrom/wavedrom/pull/337
